test/suites/integration/bsp/automatic-tipping.test.ts

```diff
@@ -15,7 +15,7 @@ describeBspNet(
       await api.file.newStorageRequest("res/whatsup.jpg", "test/whatsup.jpg", "nothingmuch-2");
       await api.wait.bspVolunteer();
 
-      await sleep(1000); // wait for the bsp to download the files
+      await api.wait.bspStoredInTxPool();
```

Comment:
Can you remove the `extrinsicPresent` call below since this does the same thing?

Reply:
Done!